<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButtonClear">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(255, 255, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButtonDiv">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 255, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="pushButtonMinus">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 255, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButtonMul">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 255, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton6">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(184, 184, 184);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButtonPoten">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(255, 255, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton7">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(184, 184, 184);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton8">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(184, 184, 184);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton5">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(184, 184, 184);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButtonDot">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(184, 184, 184);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButtonDel">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(255, 255, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>del</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton4">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(184, 184, 184);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton0">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(184, 184, 184);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton9">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(184, 184, 184);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton1">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(184, 184, 184);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton3">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(184, 184, 184);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton2">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(184, 184, 184);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButtonSgn">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(184, 184, 184);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>-/+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButtonPlus">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 255, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButtonEq">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(255, 0, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}

QPushButton:pressed{
	background-color:rgb(255, 255, 0);
	padding: 5px;
	border: 1px solid rgb(191, 191, 191);
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLineEdit" name="lineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color:rgb(184, 184, 184);
}</string>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
